.github/scripts/buildActions.sh

```diff
@@ -4,7 +4,8 @@
 # This bundles them with their dependencies into a single executable node.js script.
 
 # In order for this script to be safely run from anywhere, we cannot use the raw relative path '../actions'
-declare -r ACTIONS_DIR="$(dirname "$(dirname "$(realpath "$0")")")/actions"
+declare ACTIONS_DIR
```

Comment:
Can we just use the one liner again?

```suggestion
declare -r ACTIONS_DIR="$(dirname "$(dirname "$0")")/actions"
```

Reply:
Cool cool, ignore me! 🙈